examples/usb_host_descriptors_find_boot_keyboard.py

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 Tim Cocks for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+import array
+
+import usb
+
+import adafruit_usb_host_descriptors
+
+# lists for mouse interface indexes, endpoint addresses, and USB Device instances
+# each of these will end up with length 2 once we find both mice
+mouse_interface_indexes = None
```

Comment:
Please update the `mouse` references in this example. Let's fix it before it gets copied everywhere.

Reply:
Thank you. In the new commit, all of the mouse references are changed to keyboard and these variable names are adjusted to non-plural which is correct for this example.